apps/frontend/src/libs/components/balance-wheel-chart/libs/helpers/get-second-part-center/get-second-part-center.helper.ts

```diff
@@ -0,0 +1,7 @@
+import { AngleCoefficient } from "../../enums/enums.js";
+
+const getSecondPartCenter = (start: number, end: number): number => {
```

Comment:
What are `end` and `start`? Try to use more descriptive names

Reply:
Renamed them to `lowerBound` and `upperBound` as I use them for both left-right and top-bottom calculations.